lib/cbdocument.utils.js

```diff
@@ -125,6 +125,12 @@ function buildRefValues (indexes, data) {
         value = processValue(mpath.get(path, data))
       }
 
+      if (indexes[v].refKeyCase === 'upper') {
+        value = value.toUpperCase()
+      } else if (indexes[v].refKeyCase === 'lower') {
```

Comment:
Should `upper` and `lower` be in constants?

Reply:
probably yeah. the other places refKeyCase is used its just the string, but I'll look at updating

Reply:
yea maybe make these consts somewhere I guess?